.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "es5"
+  "trailingComma": "es5",
+  "endOfLine": "auto"
```

Comment:
Is this to avoid `CR` lint error?

Reply:
Yes had an issue siwtching form mac to window

Reply:
Nice, I had same issue with Linux too